src/components/date-time-picker/day-view/day-view.component.html

```diff
@@ -3,16 +3,24 @@
 <table class="calendar">
   <thead>
     <tr>
-      <th *ngFor="let day of weekdays" class="weekday">{{ day }}</th>
+      <th *ngFor="let day of weekdays" class="weekday" [attr.aria-label]="day">{{ day }}</th>
     </tr>
   </thead>
 
-  <tbody>
-    <tr *ngFor="let row of days">
-      <td *ngFor="let day of row" class="date-cell" [class.current]="day.today" 
-        [class.active]="day.active" [class.preview]="!day.currentMonth" 
-        (mousedown)="select(day.date)" (keyup.enter)="select(day.date)" 
-        tabindex="0">{{ day.date.getDate() }}</td>
+  <tbody role="grid">
+    <tr role="row" *ngFor="let row of days">
+      <td *ngFor="let day of row"
+          class="date-cell"
+          role="gridcell"
```

Comment:
Somehow we should indicate that this acts like a button, i.e. you can push enter to select. Since there is already a role, I'm not sure what is best other than adding a child element.